### DISCUSSION TITLE
Viewing 'My Fork'

### DISCUSSION THREAD

**Discussion by hslee2008** [community/community#15418] [Other Feature Feedback, Questions, & Ideas] [answered]
It seems like there are two ways to access my fork of a repo.

1. Go to my profile page and find the repo
2. Go to the list of forked repo and find mine

I am not absolutely sure there is another way of accessing my fork, but I have not found it yet.

Please make a button for 'My Fork' or tell me where I can find my fork.

**Comment by LinqLover** [comment]
+1. Not so long time ago, you could just print on the "Fork" button on the right upper corner of any page and then click on "view my fork". Recently, this option has been removed. Please bring it back!

**Reply by jsoref** [reply]
Indeed, I'd click <img width="78" alt="image" src="https://user-images.githubusercontent.com/2119212/165782492-e60d654e-c462-4649-a1c3-cbdb718717bd.png">, and there was a dialog with links to my forks.

Now when I click, I get this:
<img width="547" alt="image" src="https://user-images.githubusercontent.com/2119212/165782274-eed041ce-be08-4476-af2a-5bebb5bd0fc2.png">

It'd be nice if I could get to any of the repositories for which `fork already exists`.

**Reply by LinqLover** [reply]
Very precisely! Maybe this could even be implemented as a combobox similar to "edit pins" from which you could either open your fork(s) or choose to create a new one, so we do not even need to load a new `/fork` page for this special case.

![image](https://user-images.githubusercontent.com/38782922/165791135-480f21b4-ae68-4240-a321-b97f51a712f8.png)

**Comment by hslee2008** [comment]
When clicking 'fork' button, it now shows the following image if you have already created a fork before.

![image](https://user-images.githubusercontent.com/78584173/166093393-40c01849-7d75-4e4b-8eec-939478e82585.png)

I think the inconvenience of having to scroll through all the lists of forks or scrolling through your repositories are solved.

**Reply by jsoref** [reply]
That depends on how many orgs you're in. I'm in 10, so, I get this:

<img width="584" alt="image" src="https://user-images.githubusercontent.com/2119212/166128637-09ce096d-f869-43ef-b8c6-1e6558d87f30.png">

I don't want to create forks into each of my orgs just to test this behavior.

**Reply by atsju** [reply]
I confirm, If you have an organisation you can't find your fork easily.
![image](https://user-images.githubusercontent.com/4628382/166137390-9e5a65c3-d91e-4143-8db2-d68831eb6078.png)

**Reply by hslee2008** [reply]
GitHub should provide lists of your forks/organization forks when checking your forks in a repo.

**Comment by hslee2008** [comment, marked as answer]
Fixed now.

![Screenshot_20220715-223437_Brave - Nightly.png](https://user-images.githubusercontent.com/78584173/179233821-46d8367a-4ccf-44a1-9988-c53aa5a4747d.png)
